packages/api/src/schema/api-schema.yaml

```diff
@@ -3747,304 +3714,6 @@ paths:
                 $ref: "#/components/schemas/task"
         default:
           description: Error
-  /transcode:
```

Comment:
~~wow didnt even know we had this documented~~

Oh, actually this one is for the `transcode-file` task, so we should keep it. The `transcode` task was in `/asset/:id/transcode` if I'm not mistaken, which isn't documented anywhere.

Reply:
whoops yep you're right